lib/ZumoOta/src/App.cpp

```diff
@@ -83,12 +95,22 @@ void App::setup()
     Serial.print("IP Address: ");
     Serial.println(WiFi.localIP());
 
-    server.on("/", HTTP_GET, [](AsyncWebServerRequest *request){
-        request->send(200, "text/plain", "Hello, world");
-    });
-
-    server.begin();
+    webserver.begin();
+    fileManager.createUploadPage();
+    
+    String pageContent = fileManager.readUploadPage();
+    
+    if (!pageContent.isEmpty()) { 
+      
+        webserver.on("/", HTTP_GET, [pageContent](AsyncWebServerRequest *request) {

```

Comment:
the webserver config should move into your custom webserver class, or otherwise it has no real value. From app it should only be webserver.init() or similar.

Reply:
I will perform it with the next commit.